loqusdb/build_models/variant.py

```diff
@@ -201,14 +204,15 @@ def build_variant(
             ind_pos = ind_obj["ind_index"]
 
             if gq_qual:
-                gq = 0
+                gq = -1
                 if variant.QUAL:
                     gq = int(variant.QUAL)
 
             if not gq_qual:
                 gq = int(variant.gt_quals[ind_pos])
 
-            if gq_threshold and gq < gq_threshold:
+            # When gq is missing in FORMAT cyvcf2 assigns a score of -1
+            if (gq_threshold and 0 <= gq < gq_threshold) or (gq == -1 and not ignore_gq_if_unset):
```

Comment:
Looks good! I felt an urge to doublecheck that missing was indeed still -1 from cyvcf2, and indeed! Note the comment though. 😸 
https://github.com/brentp/cyvcf2/blob/541ab16a255a5287c331843d8180ed6b9ef10e00/cyvcf2/cyvcf2.pyx#L1846